### PR TITLE
added hairline views and example

### PR DESCRIPTION
**Pull request by ateliercw**
addresses #9

### CODE REVIEW

File: Pod/Classes/HairlineView/HairlineView.swift [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+// MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+// NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+// LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+// OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+// WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+//
+
+import Foundation
+
+open class HairlineView: UIView {
```
Comment (ahtierney) [inline Pod/Classes/HairlineView/HairlineView.swift]: #? do you need `@IBDesignable` here as well?

File: Pod/Classes/HairlineView/HairlineView.swift [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+    @IBInspectable open var thickness: CGFloat = CGFloat(1.0 / UIScreen.main.scale) {
+        didSet {
+            invalidateIntrinsicContentSize()
+            setNeedsUpdateConstraints()
+        }
+    }
+
+    @IBInspectable open var hairlineColor: UIColor = UIColor.darkGray {
+        willSet {
+            var red: CGFloat = 0, green: CGFloat = 0, blue: CGFloat = 0, alpha: CGFloat = 0
```
Comment (ahtierney) [inline Pod/Classes/HairlineView/HairlineView.swift]: 👍  good catch on this

File: Pod/Classes/HairlineView/HairlineView.swift [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+    }
+
+    open override func draw(_ rect: CGRect) {
+        super.draw(rect)
+        hairlineColor.setFill()
+        UIRectFill(rect)
+    }
+
+    open override func updateConstraints() {
+        defer {
```
Comment (ahtierney) [inline Pod/Classes/HairlineView/HairlineView.swift]: #pp just leave at as the last line

File: Pod/Classes/HairlineView/HairlineView.swift (hunk 1 above)
Comment (ateliercw) [inline Pod/Classes/HairlineView/HairlineView.swift]: if we want it to render visually correctly in IB, yeah. that's more work than just making it configurable in IB

**Pull request merged**